stdlib/event/time.lua

```diff
@@ -36,44 +39,54 @@ Event.Time.daily = script.generate_event_name()
 Event.register(defines.events.on_tick,
     function()
         for idx, surface in pairs(game.surfaces) do
-            local day_time = math.fmod(Event.Time.get_day_time(idx), 1)
-            local day_time_minutes = math.floor(day_time * 24 * 60)
+            if not surface.freeze_daytime then
+                local day_time = math.fmod(get_day_time(idx), 1)
+                local day_time_minutes = math.floor(day_time * 24 * 60)
```

Comment:
This is terribly nitpicky and not in all related to your changes, except that I noticed you changed indenting. Can we replace `24 * 60` with `1440`? The lua interpreter factorio is using does no JIT so that multiplication is going to happen every tick, which is just stupid.